pkg/agent/datamodel/sig_config.go

```diff
@@ -417,6 +419,13 @@ var (
 		Version:       LinuxSIGImageVersion,
 	}
 
+	SIGCBLMarinerV2Gen1ImageConfigTemplate = SigImageConfigTemplate{
```

Comment:
As you mentioned @SeanDougherty, this would ideally be SIGCBLMarinerV2ImageConfigTemplate, and the one below would be SIGCBLMarinerV2Gen2ImageConfigTemplate to pattern match with the rest.

@alexeldeib is there any risk with renaming the existing Mariner V2 Gen2 config template and fixing it up in the SigImageConfig map?

Reply:
this isn't used externally afaik, i'm fine either way

worst case we can revert it

Reply:
the key piece is the mapping of the distro name "aks-cblmariner-v2" to the image config (the string "aks-cblmariner-v2" must be the same in aks-rp)

TODO future: extract those to common data model somewhere else